### ISSUE TITLE
View: Admin-Interface

### ISSUE THREAD

**Issue by jeriox** [Schul-O-Mat/Schul-O-Mat#25]
- [ ] (Zusenden von redaktionsaccounts)
- [ ] löschen von keywords
- [ ] anlegen von neuen schulen

**Comment by labcode-de** [comment]
Wurde durch #45 und #59 ersetzt

**Comment by labcode-de** [comment]
Das haben wir doch noch garnicht oder?

**Comment by jeriox** [comment]
Ne, haben wir noch nicht. Finde ich aber ziemlich wichtig.

**Comment by labcode-de** [comment]
Function: #87

**Comment by labcode-de** [comment]
Das ist für uns, oder?

**Comment by jeriox** [comment]
Genau. Ich hatte eigentlich auch im Kopf dass da noch mehr Features stehen wie Schulcode neu generieren oder so, der ganze Kram halt den wir vielleicht mal machen müssen. Und auf dem Produktivsystem würde ich halt nur ungerne in der Datenbank rumwühlen...

**Comment by leona-ya** [comment]
Funktion dazu: #87
